src/MSALWrapper.Test/AuthFlow/AuthFlowExecutorTest.cs

```diff
@@ -218,7 +220,7 @@ public async Task HasTwoAuthFlows_Returns_Null_TokenResult()
             authFlowResultList.Add(authFlowResult2);
 
             // Act
-            var authFlowExecutor = this.Subject(new[] { authFlow1.Object, authFlow2.Object });
+            var authFlowExecutor = this.Subject(new[] { authFlow1.Object, authFlow2.Object }, this.timeoutManager);

```

Comment:
The goal of having the `Subject` Method is that a change to the creation of the subject like this one allows to to update that one method with the new arg, and you should not have to update all these calls to Subject.